s3prl/downstream/a2o-vc-vcc2020/config_ar_taco2.yaml

```diff
@@ -0,0 +1,60 @@
+runner:
+  total_steps: 10000
```

Comment:
I have a minor question irrelevant to implementation. How much time will it take for a single run under this optimization steps?

Reply:
It depends on the upstream and GPU. Smaller upstreams like APC are fast, and large models like wav2vec2 are slow. On a RTX 3090, training one model takes 3-5 hrs.

Reply:
Cool this is fast enough!